MSB4126 the specified solution configuration "Debug|x64" is invalid.

I was stepping through the instructions for building ionide-vscode for the first time, and the `dotnet restore` step failed for me:

```
C:\Users\ben\proj\oss\ionide-vscode-fsharp [master ≡]
λ  dotnet restore
C:\Users\ben\proj\oss\ionide-vscode-fsharp\ionide-fsharp-vscode.sln.metaproj : error MSB4126: The specified solution configuration "Debug|x64" is invalid. Please specify a valid solution configuration using the Configuration and Platform properties (e.g. MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") or leave those properties blank to use the default solution configuration. [C:\Users\ben\proj\oss\ionide-vscode-fsharp\ionide-fsharp-vscode.sln]
```

The obvious workaround is to invoke msbuild directly and specify Debug|Any CPU, but as it stands, the instructions for building don't work (for me, anyway).  Here's my `dotnet --info`:

```
C:\Users\ben\proj\oss
λ  dotnet --info
.NET Command Line Tools (2.1.4)

Product Information:
 Version:            2.1.4
 Commit SHA-1 hash:  5e8add2190

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.4\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.5
  Build    : 17373eb129b3b05aa18ece963f8795d65ef8ea54
```

Would a PR with a change to the defaults to "Debug|Any CPU" be accepted?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -34,6 +34,7 @@ cd ..
 
 If `dotnet restore` gives the error ` The tools version "14.0" is unrecognized`, then you need to install [msbuildtools2015][msbuildtools2015]
 
+If `dotnet restore` gives the error `error MSB4126: The specified solution configuration "Debug|x64" is invalid`, there's a good chance you have the `Platform` environment variable set to "x64".  Unset the variable and try the restore command again. 
 
 If `./build.sh` gives errors, you may need to run `./build.sh Build` one time.
 

